server/upstream/upstream-tarballs.txt

```diff
@@ -780,8 +795,10 @@ xf86-video-voodoo:httpls:http://xorg.freedesktop.org/releases/individual/driver/
 xfd:httpls:http://xorg.freedesktop.org/releases/individual/app/
 xfindproxy:httpls:http://xorg.freedesktop.org/releases/individual/app/
 xfontsel:httpls:http://xorg.freedesktop.org/releases/individual/app/
+xfsdump:ftpls:ftp://oss.sgi.com/projects/xfs/cmd_tars/
```

Comment:
Nit-picking: if we keep the list alphabetically sorted, this should be after the next line ;-)